src/App.tsx

```diff
@@ -1,24 +1,252 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, {
+  useEffect, useState, useMemo, useRef,
+} from 'react';
+import { Todo, SortType } from './Types';
 import { UserWarning } from './UserWarning';
+import { getTodos } from './todos';
+import { client } from './utils/client';
+import { ErrorMessage } from './components/ErrorMessages';
```

Comment:
Please, rename the file to `ErrorMessages` to fix this error 
![image](https://github.com/mate-academy/react_todo-app-add-and-delete/assets/99403198/17dd4574-ecf2-4a01-8f50-082cebf227d5)